src/Microsoft.TestPlatform.Extensions.TrxLogger/ObjectModel/TestMethod.cs

```diff
@@ -24,7 +24,7 @@ public TestMethod(string name, string className)
         {
             Debug.Assert(!string.IsNullOrEmpty(name), "name is null");
             Debug.Assert(!string.IsNullOrEmpty(className), "className is null");
-            this.name = name;
+            this.name = name.StartsWith($"{ClassName}.") ? name.Remove(0, $"{ClassName}.".Length) : name;
```

Comment:
@ShreyasRmsft ClassName will not even be set at this point.

Reply:
yep, need to use small 'c'
className